test_data/log_config.yml

```diff
@@ -0,0 +1,17 @@
+---
+- LOG_RECORDER: 'Ghprb'
+  LOGGERS:
+    - name: 'org.jenkinsci.plugins.ghprb.GhprbPullRequest'
+      log_level: 'ALL'
```

Comment:
From what I've been able to find in documentation, and playing with it, I think we are limited to the class-level here. https://wiki.jenkins.io/display/JENKINS/Logger+Configuration